packages/rendermime/style/index.css

```diff
@@ -47,14 +47,14 @@
 .jp-RenderedText pre .ansi-cyan-bg { background-color: #60C6C8; }
 .jp-RenderedText pre .ansi-white-bg { background-color: #C5C1B4; }
 
-.jp-RenderedText pre .ansi-bright-black-fg { color: #282C36; }
-.jp-RenderedText pre .ansi-bright-red-fg { color: #B22B31; }
-.jp-RenderedText pre .ansi-bright-green-fg { color: #007427; }
-.jp-RenderedText pre .ansi-bright-yellow-fg { color: #B27D12; }
-.jp-RenderedText pre .ansi-bright-blue-fg { color: #0065CA; }
-.jp-RenderedText pre .ansi-bright-magenta-fg { color: #A03196; }
-.jp-RenderedText pre .ansi-bright-cyan-fg { color: #258F8F; }
-.jp-RenderedText pre .ansi-bright-white-fg { color: #A1A6B2; }
+.jp-RenderedText pre .ansi-bright-black-fg { font-weight: bold; color: #282C36; }
```

Comment:
This is not an actual fix.
"Bright" colors can be used independently of "bold".
There should be a separate CSS class for "bold", but I guess this should be fixed in `ansi_up` directly?

Reply:
This has been fixed in ansi_up 3.0. Reverted this change here.